### ISSUE TITLE
Add Support for Material Dialogs

### ISSUE THREAD

**Issue by defuncart** [1SouravGhosh/flutter_inline_dialogs#2]
Presently all dialogs are rendered as Cupertino dialogs. An option could be added to render dialogs as either material, cupertino or adhering to the platform that the app is running on (i.e. iOS => Cupertino, Android => Material).

Personally I feel that _adaptive_ should be the default option. This, however, would be a breaking change as apps running on Android would then render dialogs using Material instead of Cupertino.

@1SouravGhosh I am happy to put together a solution if you like.

**Comment by 1SouravGhosh** [comment]
Yes, sure! I would like it if you help improve this package with your contribution.

**Comment by defuncart** [comment]
@1SouravGhosh Ok, I'll put together a solution when I get some spare time.

**Comment by defuncart** [comment]
@1SouravGhosh I didn't realized that @FilledStacks released his dialog manager solutions as a [package](https://pub.dev/packages/stacked_services), which has the ability to render dialogs using Material or Cupertino. I will investigate this package further.
